router/xgress_edge/dialer.go

```diff
@@ -124,7 +125,7 @@ func (dialer *dialer) Dial(destination string, circuitId *identity.TokenId, addr
 		if err != nil {
 			conn.close(false, err.Error())
 			x.Close()
```

Comment:
This one should just be a regular dial error. Terminator exists, but had trouble dialing.